plugins/talk-plugin-moderation-actions/client/components/ModerationActions.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import cn from 'classnames';
+import Tooltip from './Tooltip';
+import styles from './ModerationActions.css';
+import {Icon} from 'plugin-api/beta/client/components/ui';
+import ClickOutside from 'coral-framework/components/ClickOutside';
+import RejectCommentAction from '../containers/RejectCommentAction';
+import ApproveCommentAction from '../containers/ApproveCommentAction';
+import {Slot} from 'plugin-api/beta/client/components';
+
+export default class ModerationActions extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      tooltip: false
```

Comment:
The state should move to redux, just like in talk-plugin-author-menu and the reply box.
The current hiding/toggling behavior looks like (inconsistently) "working" because of a bug when rendering Slot elements. The logic when assigning keys to slot elements are incorrect and need to fix, when using exculde if. This causes this component to destruct and construct, thus starting with a fresh state.

Reply:
Perfect! Working on it :)